packages/sanity/src/core/form/inputs/ReferenceInput/ReferenceLinkCard.tsx

```diff
@@ -29,7 +29,14 @@ export const ReferenceLinkCard = forwardRef(function ReferenceLinkCard(
   props: ReferenceLinkCardProps & React.HTMLProps<HTMLElement>,
   ref: React.ForwardedRef<HTMLElement>,
 ) {
-  const {documentType, as: asProp, ...restProps} = props
+  const {
+    as: asProp,
+    // We exclude `documentId` from spread props to avoid passing it to the Card component
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    documentId,
```

Comment:
Would love to know if there's a more elegant way to do this!

Reply:
I think you should be able to assingn it to `_` and have eslint ignore it, e.g. `documentId: _`

Reply:
I considered this! Though I believe we need to amend our eslint config slightly to accommodate, e.g. 

```
'@typescript-eslint/no-unused-vars': [
  'warn',
  {
    varsIgnorePattern: '^_',
  },
],
```

Happy to add if you feel appropriate! Otherwise we can just leave as-is

Reply:
Since `const {foo: _, bar: _, ...rest} = …` will yell at you for redeclaring consts, will that rule allow omitting more than one property? Maybe we could configure it to be `_ignore_${propertyName}` (or maybe that's too much?).

There's also [no-unused-vars#ignorerestsiblings](https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings), which is more specific to this particular case.